### PR TITLE
AME-3223 change ad account ID to pixel ID

### PR DESCRIPTION
**Pull request by PinkBananaCheese**
## 💸 TL;DR

Updates the template references of "advertiserId" and "Ad Account ID" to "pixelId" and "Pixel ID", and updates the template screenshot

## 🧪 Testing Steps / Validation
- Updated locations on the template fields
Screenshot 2023-11-13 at 10 15 40 AM

- Test output
Screenshot 2023-11-13 at 10 18 29 AM

### CODE REVIEW

File: template.tpl [hunk 1]
```diff
@@ -49,7 +49,7 @@ ___TEMPLATE_PARAMETERS___
         "type": "REGEX"
       }
     ],
-    "displayName": "Ad Account ID",
```
Comment (woodchuckgym) [inline template.tpl:52]: I think the answer is "it's out of scope for now", but we're not changing the prefix on the pixel ID, right?  That is, we're still good-to-go on line 47 with the `(t2_|a2_)[a-z0-9]+` regex, right?

**Review by woodchuckgym — APPROVED** [review]

File: template.tpl (hunk 1 above)
Comment (creativefunction13) [reply template.tpl:52]: I think the plan is to update the prefix for pixels.  We will always support both `t2` and `a2` as those ids will likely never change on old websites.  New pixels created once we have the full creation process will have a different prefix.

Comment (PinkBananaCheese) [reply template.tpl:52]: correct! no changes to any IDs at the moment. And then as @creativefunction13 noted, when we do, it'll just be for new IDs, so the regex would be updated to also support whatever new scheme that ends up being while continuing to support the old ones

**Comment by PinkBananaCheese** [comment]
I had held the PR due to code-freeze timing. Needed to rebase and update the screenshot

**Review by woodchuckgym — APPROVED** [review]
All your base are belong to us

**Pull request merged**